src/main/public/stylesheets/components/_form.scss

```diff
@@ -65,3 +65,7 @@ select {
     width: 50%;
   }
 }
+
+.second-text-input {
+  margin-top: 15px;
```

Comment:
I can see a lot of places where we have margin-*: 15px. Why don't we add some sccs var for this value?

NOT FOR THIS PR